apps/nsqadmin/main.go

```diff
@@ -47,6 +47,8 @@ var (
 	httpClientTLSKey                = flagSet.String("http-client-tls-key", "", "path to key file for the HTTP client")
 
 	allowConfigFromCIDR = flagSet.String("allow-config-from-cidr", "127.0.0.1/8", "A CIDR from which to allow HTTP requests to the /config endpoint")
+	aclHttpHeader = flagSet.String("acl-http-header", "X-Forwarded-User", "HTTP header to check for authenticated users (default: X-Forwarded-User)")
```

Comment:
I don't think you have to note the default in the help string yourself, if you specify the default in the flag declaration as you have done. For example, `allow-config-from-cidr` is described like so in `nsqadmin --help`:

```
  -allow-config-from-cidr string
    	A CIDR from which to allow HTTP requests to the /config endpoint (default "127.0.0.1/8")
```

(We only manually note the default in help strings when we set the default in the flag declaration.)